packages/lib/shared/components/modals/PoolCreationModalFooter.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+
+import { ModalFooter, Text, HStack } from '@chakra-ui/react'
+
+export function PoolCreationModalFooter({ onReset }: { onReset: () => void }) {
```

Comment:
is this for tests only?

Reply:
is something we should discuss with Pon.

If we dont offer reset button and user wants to abandon after finishing create pool step, they would have to switch browsers or clear local storage in dev tools

Reply:
They can go back and click the edit button (it tries to reset the form), unfortunately it does not completely work as of right now.

<img width="735" alt="Screenshot 2025-06-09 at 09 05 55" src="https://github.com/user-attachments/assets/fa78c79c-dd7e-49bc-80ee-0c07a4a966ac" />

Another issue when using local storage is what happens when a new version drops that changes the schema of the saved data. I had issues when I started testing because it will try to use old data.